src/components/swipe.js

```diff
@@ -163,6 +172,7 @@ export default function (Glide, Components, Events) {
      * @return {Void}
      */
     bindSwipeMove () {
+      Binder.on(MOVE_EVENTS, Components.Html.wrapper, this.preventDefaultMove, false)
```

Comment:
Binder can store only one listener callback per event (look at its [source](https://github.com/glidejs/glide/blob/master/src/core/event/events-binder.js#L26)).

That means that `Binder.on` method one line below will replace both `this.listeners['touchmove']` and `this.listeners['mousemove']`. This in result makes it impossible to remove `preventDefaultMove` event listeners. During `Binder.off` `this.listeners` will store only throttled `this.move(event)` and only it will be passed to `el.removeEventListener`.